edward/inferences/inference.py

```diff
@@ -194,6 +210,16 @@ def initialize(self, n_iter=1000, n_print=None, scale=None, logdir=None,
 
     if logdir is not None:
       self.logging = True
+
+      if logrun is None:
+        # Set default to timestamp
+        logrun = datetime.strftime(datetime.utcnow(), "%Y%m%dT%H%M%S")
```

Comment:
Good standard! What's the T? 

Also, do you think adding `_` inbetween would make the directory name too long?